python/py-xraylarch/Portfile

```
@@ -28,14 +28,14 @@ long_description    Larch is a open-source library and set of applications for p
 homepage            https://xraypy.github.io/xraylarch
 
 # Find the latest at https://pypi.org/project/xraylarch/#files
-master_sites        https://files.pythonhosted.org/packages/ff/4a/cf7f494aaa3ea1ba15adf5806cef0ac5136affc7e8f7c2048999fa4e28d9/
+master_sites        https://files.pythonhosted.org/packages/69/ee/25315817d35756a170df5c022f3d2ec550a599df927a6b1e02b3079dbcef/
 distname            xraylarch-${version}-py3-none-any
 
-checksums           rmd160  0736ee2e7d6ba40c8c55dc7111cb3c3067454fd2 \
-                    sha256  92d558df4c8302a277ead4ad597920285abf3b7e69ca9ab13873e8dc319e452a \
-                    size    21056444
+checksums           rmd160  52774b232a776f0b4a5ec35b7de55dbbdefc8b61 \
+                    sha256  6f24b5118113c6dce65a78067d821204e43f63a34d23e6bb3378f771e1fd97c4 \
+                    size    21125031
 
-python.versions     37 38 39 310
+python.versions     38 39 310
```

Current default Python version in MacPorts is 3.11.  Also 3.12 has just been added.  Is it possible to add those 2 as well?

```suggestion
python.versions     38 39 310 311 312
```

---

I can't add python 3.11, as some of the dependencies aren't available in 3.11. `py-fabio` is one specific example.

I admit that I'm the maintainer of those, so we could pause this PR until I get the requirements up to date, but there's no guarantee that all the upstream sources are 3.11-compatible. I would have to check.

Regarding 3.12, the upstream source for this package makes no mention of Python 3.12. I don't know what is Macports' standard practice. Do we assume that 3.12 will work, or do we wait for the package authors to say so explicitly?

---

I propose you merge this PR, and I'll promise to work on getting 3.11 versions into all the ports I maintain.